scripts/install_R_source.sh

```diff
@@ -11,6 +11,7 @@
 set -e
 
 R_VERSION=${1:-${R_VERSION:-"latest"}}
+PURGE_BUILDDEPS=${PURGE_BUILDDEPS=-"TRUE"}
```

Comment:
I think bools in env vars should be `"true"` or `"false"`, not `"TRUE"` or `"FALSE"`.

Reply:
I _think_ env vars are untyped so 1 and "1" and "TRUE" and "True" and "true" are all the same.  Unless, of course, the receiving end makes a difference when it parses.  Otherwise the check is commonly 'set or not set'.

Reply:
Is that a common practice in shell script?
I saw in bash:

```bash
$ if "true"; then echo "true"; fi
true

$ if "false"; then echo "true"; fi

$ if "1"; then echo "true"; fi
bash: 1: command not found

$ if "TRUE"; then echo "true"; fi
bash: TRUE: command not found

$ if "True"; then echo "true"; fi
bash: True: command not found
```

Reply:
right, but as you I did this a string comparison since booleans in bash always confuse me.  Anyway I've changed these to lower-case now.  Thanks for catching this!  let me know if you'd recommend further changes here.